npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -32,6 +32,26 @@ def self.required_files_message
 
       private
 
+      # Overridden to pull any yarn data or plugins which may be stored with Git LFS.
+      def _clone_repo_contents(target_directory:)
```

Comment:
At least at one time in history, methods like this one in `Dependabot::FileFetchers::Base` were not meant to be overridden:

https://github.com/dependabot/dependabot-core/blob/6a9c745252f7ff8389751b58133c851df6207b79/common/lib/dependabot/file_fetchers/base.rb#L176-L178

This would be the first instance of a file fetcher subclass overriding it. Could we override public method `#clone_repo_contents` instead?